### PR TITLE
Upload progress feature for explore submissions

### PR DESCRIPTION
**Pull request by sbibek**
Added support for submissions on explore page. Error handlers in place so that errors won't break existing features. Also fix avoids note on explore section as there are none.

### CODE REVIEW

File: scripts/leetcode.js [hunk 1]
```diff
@@ -600,22 +606,40 @@ function startUploadCountDown() {
...
-  elem = document.getElementById('leethub_progress_anchor_element')
-  if (elem !== undefined) {
-    elem = document.createElement('span')
-    elem.id = "leethub_progress_anchor_element"
+
+/* we will need specific anchor element that is specific to the page you are in Eg. Explore */
+function insertToAnchorElement(elem){
+  if(document.URL.startsWith("https://leetcode.com/explore/")) {
+    // means we are in explore page
+    target = document.getElementsByClassName('action')[0].getElementsByClassName('row')[0].getElementsByClassName('col-sm-6')[1]
```
Comment (QasimWani) [inline scripts/leetcode.js:614]: assumption ur making here is all these fields are available. u need to do null check for this, unless this is guaranteed to be present within leetcode. plz verify the ubiquity of this structure @sbibek

**Review by QasimWani — CHANGES_REQUESTED** [review]

**Review by sbibek — COMMENTED** [review]
Cardinality checks for all the elements used are now made.

File: scripts/leetcode.js [hunk 2]
```diff
@@ -608,25 +611,34 @@ function startUploadCountDown() {
...
+    }
   }
 }
 
 /* start upload will inject a spinner on left side to the "Run Code" button */
 function startUpload() {
   try {
     elem = document.getElementById('leethub_progress_anchor_element')
-    if (elem !== undefined) {
+    if (!elem) {
```
Comment (QasimWani) [inline scripts/leetcode.js:641]: this seems the opposite to what was done before. `!elem` tells me that if that element is not present, create the spinner. But previously, you had it as if the element was present, create a spinner.
reason? @sbibek

**Review by QasimWani — COMMENTED** [review]
plz see comments.

**Review by sbibek — COMMENTED** [review]
It's the fix for the unsound logic that was there but worked due to the reason null != undefined

**Review by QasimWani — APPROVED** [review]

**Pull request merged**